lib/lingo/pegcode/pegcompiler.flow

```diff
@@ -467,3 +470,38 @@ removePegActions(pa : Parsing) -> Parsing {
 		LingoValue(value): pa;
 	}
 }
+
```

Comment:
Move these to a separate file. Maybe one of the existing helpers can be used to do this, but otherwise OK.